### ISSUE TITLE
Guidelines: How to enqueue successfullly many jobs in SMRT web portal

### ISSUE THREAD

**Issue by damianosmel** [PacificBiosciences/SMRT-Analysis#179]
Dear SMRT Portal team or expert users!

In our group, we are have started to run the SMRT portal web service (v. 2.3.0) for genome assembly of prokaryotes on a 1TB RAM machine with 64 processors. 
We know how to run one job (de novo assembly) per time from the web service, but we would like to submit several jobs via the web service in order to maximize the machine usage, e.g. over a weekend.

However, currently when we try to start 2 or more jobs simultaneously, only the first job finishes successfully, the others do not finish.
Therefore, we would be very grateful for some suggestions regarding commands that would allow us to successfully run multiple jobs sequentially, either from the web service or via the command line. Likely there is a functionality that will allow us to do that and we would appreciate input from more experienced users. 

Thank you in advance for your time!

**Comment by mjhsieh** [comment]
On the command line side, it should be easy to do it. Have something like this batch file, say batch.txt:
```
bash jobdir1/job.sh
bash jobdir2/job.sh
```
Then just do this to run two jobs sequentially:
```bash batch.txt```

On the web service side, you might be able to do something in this file (Make sure it's just in one line.) ```$SMRT_TOPDIR/analysis/etc/cluster/BASH/interactive.tmpl```

**Comment by damianosmel** [comment]
Hi,
Thank you for your help and sorry for this slow answer.

My colleague and I have achieved to follow the first of your two suggested ways!

A quick summary of what we did (may be helpful to someone else):
1) Create the jobs from SMRT web service (=> it will create the root job output folder inside the $SMRT_TOPDIR/jobs folder and the settings.xml)
2) Create, by ourselves, the input.xml, by comparing to the respective input.xml of an already SMRT job run.
3) Abjust the job.sh, from an already SMRT job run, to match our new job input and output arguments.
4) Then, as it is already suggested, create a batch.txt and input the bash commands to run each new job.sh.
5) Run the batch.txt
